sdk/python/jobs/automl-standalone-jobs/automl-forecasting-task-bike-share/auto-ml-forecasting-bike-share.ipynb

```diff
@@ -203,18 +212,82 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   },
    "source": [
-    "We will upload the dirrectory with the test set so that further we will be able to run the inferencing on the batch endpoint."
+    "To create data input from TabularDataset created using V1 sdk, set the `type` to `AssetTypes.MLTABLE`, `mode` to `InputOutputModes.DIRECT` and `path` to the following format `azureml:<tabulardataset_name>` or `azureml:<tabulardataset_name:<version>`(in case we want to use specific version of the registered dataset)."
```

Comment:
Are we using TabularDataset created in v1? The csv file is located in this folder.

Reply:
Yes. We are using tabular dataset created in V1.